draft-ietf-tls-esni.md

```diff
@@ -427,13 +439,16 @@ extensions MUST be ordered consecutively in ClientHelloInner. The list of outer
 extensions, OuterExtensions, includes those which were removed from
 EncodedClientHelloInner, in the order in which they were removed.
 
-Finally, EncodedClientHelloInner is serialized as a ClientHello structure,
-defined in Section 4.1.2 of {{RFC8446}}. Note this does not include the
-four-byte header included in the Handshake structure.
+Finally, the client sets the `padding` field to a byte string whose contents
+are all zeros. {{padding}} describes a recommended padding scheme.
```

Comment:
```suggestion
Finally, the client sets the `padding` field to a byte string of length N whose
contents are all zeros. {{padding}} describes a recommended padding scheme.
```

Reply:
Done, although tweaked slightly since {{padding}} is merely recommended, not required.